src/main/java/com/faforever/client/preferences/LiveReplaySearchPrefs.java

```diff
@@ -0,0 +1,29 @@
+package com.faforever.client.preferences;
+
+import com.faforever.client.domain.api.FeaturedMod;
+import com.faforever.commons.lobby.GameType;
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class LiveReplaySearchPrefs {
+
+  private final BooleanProperty hideModdedGames = new SimpleBooleanProperty(false);
+  private final BooleanProperty hideSingleGames = new SimpleBooleanProperty(false);
+  private final BooleanProperty onlyGamesWithFriends = new SimpleBooleanProperty(false);
+  private final BooleanProperty onlyGeneratedMaps = new SimpleBooleanProperty(false);
+
+  private final ListProperty<GameType> gameTypes = new SimpleListProperty<>(FXCollections.observableArrayList());
+  private final ListProperty<FeaturedMod> modName = new SimpleListProperty<>(FXCollections.observableArrayList());
+
+  private final StringProperty playerName = new SimpleStringProperty("");
+
+}
```

Comment:
Unfortunately we cannot use lombok getter and setter on these javafx property beans as they will not serialize and deserialize properly.

This is because the getter and setter are produced for the property object itself rather than the underlying representation, thus jackson will serialize the property object and not the value. To this end we follow javafx bean convention of 
```
T get*()
void set*(T)
Property<T> *Property()
```

You can see this pattern in the other pref objects

Reply:
Got rid of lombok and manually crfaeted methods.